src/InsertionsClient/Common/Constants/InsertionConstants.cs

```diff
@@ -18,5 +18,7 @@ internal static class InsertionConstants
             "VS.Tools.Net.Core.SDK.Resolver",
             "VS.Tools.Net.Core.SDK.x86"
         };
+
+        internal const string DefaultNugetFeed = "https://pkgs.dev.azure.com/devdiv/_packaging/VS-CoreXtFeeds/nuget/v3/index.json";

```

Comment:
consider placing this in app.config to avoid requiring recompilation in cases where the URL must be changed.